plugin/pkg/admission/limitranger/admission_test.go

```diff
@@ -30,14 +30,17 @@ import (
 	"k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset/fake"
 )
 
-func getResourceList(cpu, memory string) api.ResourceList {
+func getResourceList(cpu, memory, storage string) api.ResourceList {
```

Comment:
I originally wanted to do something like that but thought I was being too lazy, so I edited every single func call ...